### ISSUE TITLE
Unity-CM: Make Nightly access MC through the Shared Service (via httpd, then private ALB)

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-cs#308]
Depends on #352, #354 

![Image](https://github.com/unity-sds/unity-cs/assets/47542238/f8faf2ab-76a5-48aa-a8f2-ff20e4844e92)

Now that the MC is behind Cognito, we will need a methodology to pass credentials so that the nightly can authenticate.  This might be with secure SSM params, or perhaps done all via the URL.  Research what way is best and secure.

**Comment by galenatjpl** [comment]
Blocked by https://github.com/unity-sds/cfn-ps-jpl-unity-sds/issues/14

**Comment by mike-gangl** [comment]
I don't think this is blocked anymore. but i'm not sure it's _done_ either.

**Comment by galenatjpl** [comment]
@mike-gangl You are correct that this is not blocked anymore.  All of the pieces are in place.  They just need to be configured properly, then it should prove the test works.  Once configured, and tested, this ticket can be close.
